consensus/ssz/src/decode/impls.rs

```diff
@@ -271,6 +272,20 @@ impl<T: Decode> Decode for Option<T> {
     }
 }
 
+impl<T: Decode> Decode for Arc<T> {
```

Comment:
I don't think I'm opposed to this, but an interesting thing to note is that generally you use an `Arc<T>` to maintain a reference to the same `T`. However if you do an encode/decode cycle with these impls then you'll end up with a different `T`.

I'm not sure how much awkwardness having this impl saves, but I think it's worth considering that it's *potentially* a bit against the grain of `Arc`.

Reply:
If this is just for the `SyncCommittee` on `BeaconState` then we can probably just drop these impls, I only just saw that.